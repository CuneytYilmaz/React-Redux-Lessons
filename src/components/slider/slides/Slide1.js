import React from 'react';

const Slide1 = () =>
  <div className="Slider__slide">
    <h1 className="Slider__title Slider__title--main">
      Sen de yemek yerken, ne izleyeceğine karar veremiyor musun?
    </h1>
    <p>
      Sen yemeğini yerken ne izleyeceğini düşünme ya da izleyecek bir şeyler ararken yemeğini
      soğutma diye ses ile de kontrol edebileceğin bir sistem geliştirdik.
    </p>
  </div>;

export default Slide1;
