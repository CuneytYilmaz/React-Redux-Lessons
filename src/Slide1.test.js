import React from 'react';
import { shallow } from 'enzyme';

import Slide1 from './components/slider/slides/Slide1';

it('renders correctly', () => {
  const wrapper = shallow(<Slide1 />);

  expect(wrapper).toMatchSnapshot();
});
