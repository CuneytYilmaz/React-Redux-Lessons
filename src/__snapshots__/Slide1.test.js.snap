// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row Slider__slide"
>
  <div
    className="col-md-6"
  >
    <h1
      className="Slider__title Slider__title--main"
    >
      Sen de yemek yerken, ne izleyeceğine karar veremiyor musun?
    </h1>
    <p
      className="u-gap-bottom"
    >
      Sen yemeğini yerken ne izleyeceğini düşünme ya da izleyecek bir şeyler ararken yemeğini soğutma diye ses ile de kontrol edebileceğin bir sistem geliştirdik.
    </p>
    <Button
      primary={true}
      secondary={false}
    >
      Kullanmaya Başla!
    </Button>
  </div>
  <div
    className="col-md-6 u-text-center"
  >
    <img
      alt="Sen de yemek yerken, ne izleyeceğine karar veremiyor musun?"
      className="u-inline-block"
      height="378"
      src="slider-image-1.svg"
      width="400"
    />
  </div>
</div>
`;
